fs/config/crypt.go

```diff
@@ -144,6 +148,9 @@ func Decrypt(b io.ReadSeeker) (io.Reader, error) {
 			if usingPasswordCommand {
 				return nil, errors.New("using --password-command derived password, unable to decrypt configuration")
 			}
+			if usingEnvPassword {
```

Comment:
This can be just `if envpw != ""` can't it? Then that removes all the other changes?

Reply:
`envpw` is not in the scope